sdcm/ec2_client.py

```diff
@@ -328,14 +332,15 @@ def create_spot_instances(self, instance_type, image_id, region_name, network_if
 
         LOGGER.info('Spot instances: %s', instance_ids)
         for ind, instance_id in enumerate(instance_ids):
-            self.add_tags(instance_id, [{'Key': 'Name', 'Value': 'spot_{}_{}'.format(instance_id, ind)}])
+            self.add_tags(instance_id, tags_list + [{'Key': 'Name', 'Value': 'spot_{}_{}'.format(instance_id, ind)}])
```

Comment:
why are you messing with the logic here ?

Reply:
I changed the logic because the function got from the user list of tags (`tags_list`) and not used this variable...